JS/Planes/Plane.js

```diff
@@ -1,28 +1,22 @@
 class Plane {
-
-    constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity) {
-        this.model = model;
-        this.maxSpeed = maxSpeed;
-        this.maxFlightDistance = maxFlightDistance;
-        this.maxLoadCapacity = maxLoadCapacity;
+    constructor(aircraftModel, speedLimit, distanceLimit, loadCapacity) {
```

Comment:
This constructor has too many arguments. It would be a good idea to pass a single object to a constructor. This object will have properties: model, maxSpeed, etc.